pkgs/development/tools/omnisharp-roslyn/updater.sh

```diff
@@ -1,19 +1,8 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i bash -p curl jq common-updater-scripts nuget-to-nix
+#! nix-shell -I nixpkgs=./. -i bash -p curl jq common-updater-scripts
```

Comment:
Shouldn't this be `-I nixpkgs=../../../..`?
```suggestion
#! nix-shell -I nixpkgs=../../../.. -i bash -p curl jq common-updater-scripts
```

Reply:
IIRC the maintainer script will depend on running the update script from nixpkgs root. That's what r-ryantm uses, so compatibility is definitely something we want.

Reply:
Oh ok, ignore all the other comments then

Reply:
Don't think `-I nixpkgs=./.` happens by default, using the current nixpkgs checkout is still valuable id say. Other comments still seem useful to me, just took some liberties with applying them :)